hashicorp/boundary-ui
# Alias clear modal

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-12928)

## Description

This PR adds a delete modal with an extra option `clear` so users can clear aliases instead of deleting it

## Screenshots (if appropriate):

https://github.com/hashicorp/boundary-ui/assets/15043878/8df848a7-4cc3-4d0b-952f-3c5210025f84


## How to Test

1. using boundary controller, create a few aliases 
2. After successful creation, you can see the aliases pop up in /aliases view 
3. click on the `...` icon on the list table, you should see `edit` and `delete` options
4. click on delete. The hds modal should show three options(delete, clear and cancel) if there's a target associated with the resource
5. If there's no target associated, you should only see two options - delete and cancel 





:technologist: [Admin preview](PATH_TO_FEATURE)

:desktop_computer: [Desktop preview](PATH_TO_FEATURE)

## Checklist:
